pkg/controller/vmnetcfg/controller.go

```diff
@@ -136,13 +146,7 @@ func (h *Handler) Allocate(vmNetCfg *networkv1.VirtualMachineNetworkConfig, stat
 			)
 
 			// Update IPPool status
-			ipPoolNamespace, ipPoolName := kv.RSplit(nc.NetworkName, "/")
 			if err := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
```

Comment:
When conflict in `RetryOnConflic`, should re-get the `ipPool` from cache like above? current var ipPool may have been out-dated
``` 
ipPool, err := h.ippoolCache.Get(ipPoolNamespace, ipPoolName)
```

The original code is re-getting `ipPool`, what's the consideration to remove the code?

Reply:
Yes, you're right. It should be inside the `RetryOnConflict` block. But I just realized we no longer need the explicit retry, as it was stale when internal caches were introduced.